app/services/EmploymentSessionService.scala

```diff
@@ -413,7 +413,7 @@ class EmploymentSessionService @Inject()(employmentUserDataRepository: Employmen
       priorDataResponse <- getPriorData(request.user, taxYear)
     } yield {
       if (optionalCya.isRight) {
```

Comment:
How about this instead:
```if (optionalCya == Right(None)) logger.info(s"[EmploymentSessionService][getAndHandle] No employment CYA data found for user. SessionId: ${request.user.sessionId}")```

Reply:
There are many problems with this module. I did not introduce this code. I fixed many warnings, but I cant fix it all. :)